consensus/src/epoch_manager.rs

```diff
@@ -77,11 +88,14 @@ pub struct EpochManager {
     network_sender: ConsensusNetworkSender,
     timeout_sender: channel::Sender<Round>,
     txn_manager: Arc<dyn TxnManager>,
-    state_computer: Arc<dyn StateComputer>,
+    state_computer: Option<Arc<dyn StateComputer>>,
```

Comment:
why this needs to be optional? we should name it ordering_state_computer and another commit_state_computer

Reply:
resolved